src/components/FormField/index.tsx

```diff
@@ -59,7 +59,7 @@ const FormField = React.forwardRef<HTMLInputElement, FormFieldProps>(
     if (errorText?.message) {
       helperText = errorText.message;
     }
-    const ariaLabelledby = helperText ? `${id}-helperText` : "";
+    const ariaDescribedby = helperText ? `${id}-helperText` : null;
```

Comment:
This is a bigger issue. `aria-labeledby` is the incorrect attribute and it should really be `aria-describedby`.